### PR TITLE
chore: update interface of Sparkline chart

### PR DESCRIPTION
**Pull request by burbuli8ra**
This PR is required for issue [Cloud Frontend #3194](https://github.com/netdata/cloud-frontend/issues/3194)

#### Solution
Update `Sparkline` chart with:
* Remove chart's stroke
* Remove opacity of chart's background color
* Update appearence and position  of chart's name
* Update appearence and position of chart's latest value
* Remove chart's border and add round its corners

### CODE REVIEW

File: src/components/hocs/withFullscreen.js [hunk 1]
```diff
@@ -3,20 +3,16 @@ import Flex from "@netdata/netdata-ui/lib/components/templates/flex"
 import Layer from "@netdata/netdata-ui/lib/components/templates/layer"
 import { useAttributeValue } from "@/components/provider"
 
-export default Component => {
```
Comment (novykh) [inline src/components/hocs/withFullscreen.js:6]: Why is this change here?

Comment (burbuli8ra) [reply src/components/hocs/withFullscreen.js:6]: It's irrelevant to this PR!
Due to [this bug](https://github.com/netdata/cloud-frontend/issues/3193) that I encountered while doing the update, I opened this file (among others) to do a tiny research for its root and my IDE's linter was giving me a "warning" for redundant const `FullscreenComponent`.
If you want me to change it back please say so! 😄

Comment (burbuli8ra) [reply src/components/hocs/withFullscreen.js]: Kept it as it was previously! 😄

**Review by novykh — APPROVED** [review]
